homeassistant/components/linear_garage_door/coordinator.py

```diff
@@ -58,7 +58,7 @@ async def update_data(linear: Linear) -> dict[str, Any]:
             for device in self._devices:
                 device_id = str(device["id"])
                 state = await linear.get_device_state(device_id)
-                data[device_id] = LinearDevice(device["name"], state)
+                data[device_id] = LinearDevice(str(device["name"]), state)
```

Comment:
Why this

Reply:
Pylance was complaining
```
Argument of type "list[str] | str | Any" cannot be assigned to parameter "name" of type "str" in function "__init__"
  Type "list[str] | str | Any" cannot be assigned to type "str"
    "list[str]" is incompatible with "str"
```

Reply:
Oh that's because of the typing, meh, I don't like this

Reply:
Just drop the cast and ignore the warning?